lib/src/feedback_widget.dart

```diff
@@ -169,27 +172,39 @@ class FeedbackWidgetState extends State<FeedbackWidget>
                   },
                 ),
               ),
-              if (widget.isFeedbackVisible)
+              // only display if feedback is visible or this widget is still
+              // animating out
+              if (widget.isFeedbackVisible || !animation.isDismissed)
```

Comment:
This could probably be simplified to just `if (!animtion.isDismissed)`. In fact, you could replace `bool isFeedbackVisible` with `bool get isFeedbackVisible => !animationDismissed` (this will be slightly more expansive, it'll return true when the feedback is displayed AND while it's still animating out).

All that said, I think either works well and this way it's a little more obvious what the code is doing.